contract/contracts/PixelAvatars.sol

```diff
@@ -5,13 +5,15 @@ import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/utils/StringsUpgradeable.sol";
+import "./@eip2981/ERC2981ContractWideRoyalties.sol";
 
 /// @author Developer DAO
```

Comment:
Would it make sense to add `/// @author Developer DAO` to all files you edited?

Reply:
@noahliechti 

For the ERC2981 was a code copied from a repo. 

But what are your thinks if adding and @author tag on a code that wasn't originally created here?